scala-package/mxnet-demo/scala-demo/pom.xml

```diff
@@ -9,6 +9,13 @@
     <name>MXNet Scala Demo</name>
     <packaging>pom</packaging>
 
+    <repositories>
```

Comment:
this could be misleading to users, when they modify the demo code in master and try to run, but you are changing it to always pick from snapshot repo. Can you selectively only do this only for CI test?

Reply:
Good point. Done in : 1cbc59d8f87dff3fa3996916c18b95aeb215200c